### PR TITLE
config: cut over from 4.4 to 4.5 for master

### PR DESCRIPTION
**Pull request by stevekuznetsov**
Signed-off-by: Steve Kuznetsov <skuznets@redhat.com>

Generated with:

```sh
#!/bin/bash

set -o errexit
set -o nounset
set -o pipefail

sed -i 's/base-4-4/base-4-5/g' ci-operator/config/openshift/images/openshift-images-master.yaml
sed -i 's/current-release=4.4/current-release=4.5/g' ci-operator/jobs/infra-periodics.yaml
sed -i '/future-release=4.5/d' ci-operator/jobs/infra-periodics.yaml
sed -i -e 's/,latest//' -e 's/4.5.0/4.5.0,latest/' -e '/mapping_origin_4_5/ a \	BASE=quay.io/openshift/origin- VERSION=4.6 TAG=4.6,4.6.0 ../../../hack/mirror-file > mapping_origin_4_6' core-services/image-mirroring/openshift/Makefile
make -C core-services/image-mirroring/openshift image-mirror-files

docker pull registry.svc.ci.openshift.org/ci/config-brancher:latest
docker pull registry.svc.ci.openshift.org/ci/ci-operator-prowgen:latest
docker pull registry.svc.ci.openshift.org/ci/determinize-prow-jobs:latest

docker run -v "$( pwd )/ci-operator:/ci-operator:z" registry.svc.ci.openshift.org/ci/config-brancher:latest --config-dir /ci-operator/config --current-release=4.4 --bump-release=4.5 --future-release=4.5 --future-release=4.6 --confirm
docker run -v "$( pwd )/ci-operator:/ci-operator:z" registry.svc.ci.openshift.org/ci/ci-operator-prowgen:latest --from-dir /ci-operator/config --to-dir /ci-operator/jobs
docker run -v "$( pwd )/ci-operator/jobs:/ci-operator/jobs:z" registry.svc.ci.openshift.org/ci/determinize-prow-jobs:latest --prow-jobs-dir /ci-operator/jobs
```

### CODE REVIEW

**Comment by stevekuznetsov** [comment]
@smarterclayton can you please review the image mirror changes

File: core-services/prow/02_config/_plugins.yaml [hunk 1]
```diff
@@ -173,13 +173,7 @@ bugzilla:
...
       validate_by_default: true
     openshift-4.4:
-      dependent_bug_states:
-      - status: MODIFIED
-      - status: ON_QA
-      - status: VERIFIED
-      - status: RELEASE_PENDING
-      - resolution: ERRATA
-        status: CLOSED
+      dependent_bug_target_release: 4.5.0
```
Comment (eparis) [inline core-services/prow/02_config/_plugins.yaml]: shouldn't this have `validate_by_default`

Comment (stevekuznetsov) [reply core-services/prow/02_config/_plugins.yaml]: Yes

Comment (stevekuznetsov) [reply core-services/prow/02_config/_plugins.yaml]: It does, two lines down :)

**Comment by eparis** [comment]
/hold
for EOD

**Comment by stevekuznetsov** [comment]
Rebased, landing

**Comment by smarterclayton** [comment]
For next time you need to escape the `.` in the left side of the regex - `4-4` was turned into `4.5`.

**Comment by smarterclayton** [comment]
Fixed the sed to be correct (4-4 -> 4-5).  Always escape `.` in sed left side

**Pull request merged**